src/App.tsx

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from "react";
 import { getCompliments, getAdvice } from "./ApiCalls";
-import "./App.css";
-import { Button } from "./components/Button/Button";
-import { AppContainer } from "./components/AppContainer/AppContainer";
+import { Navigation } from "./components/Navigation/Navigation";
+import { Routes } from "./components/Routes/Routes";
 
-interface Props {}
+
+
+
+
+interface Props { }
 interface State {
   compliment: string;
   advice: string;
 }
 
 class App extends Component<Props, State> {
-  state: State = {
-    compliment: "",
-    advice: "",
-  };
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      compliment: "",
+      advice: "",
+    };
+  }
 
   componentDidMount() {
```

Comment:
Nice work separating your network requests into their own file.  I'm curious though, instead of having to `setState` twice, could we use something like `Promise.all()` for both GETs and then just `setState` once?  Also remember to include a `.catch()`!